sdk/go/common/workspace/plugins.go

```diff
@@ -995,6 +1004,30 @@ type PluginSpec struct {
 	Checksums map[string][]byte
 }
 
+func NewAnalyzerPlugin(name tokens.QName, version *semver.Version) PluginSpec {
+	return PluginSpec{
+		Name:    strings.ReplaceAll(string(name), tokens.QNameDelimiter, "_"),
+		Kind:    apitype.AnalyzerPlugin,
+		Version: version,
+	}
+}
+
+func NewConverterPlugin(name string, version *semver.Version) PluginSpec {
```

Comment:
Should these functions live here? `NewProvider` seems conspicuously absent, for instance? Also I'm minded to suggest `New*PluginSpec` as the names since these are constructors, but that's a nit.

Reply:
Eh, I thought they would be convenient for a bit while refactoring, but I don't think they are all that useful.  Removed them.